README.md

```diff
@@ -61,22 +80,42 @@ This project consists of two parts, a client web app in `client/`, and a web ser
         ```bash
         # From project root
         cd client/
-        node run serve
+        npm run serve
         ```
-    
 
+5. Opening app:  
+Open the client web address in your browser. The address will be listed in the terminal used for starting the client.
+      
+If you wish to remove excessive warnings about source maps from the dev console in your browser, then disable JavaScript source maps.  
+To do this in Chrome go to dev tools `Settings > Preferences > Sources` and disable `Enable JavaScript source maps`.
 
-## Contacts
 
-Maintainer: Luke Parkinson [@LukeParky](https://github.com/LukeParky/)
-[luke.parkinson@canterbury.ac.nz](mailto:luke.parkinson@canterbury.ac.nz)
+### Deploying application to production deployment
+This is a work in progress. We are using nginx and docker to deploy our application.
+Please see [this issue](https://github.com/LukeParky/gri-earthquake/issues/9) for progress details.
+
+### Repository structure
+```tree
+gri-earthquake
+└───client # files used in the client-side application, run in the user's browser  
```

Comment:
I like how you've illustrated the folder structure - both her and in the child readme's.